docs/install/server-infrastructure.md

```diff
@@ -10,6 +10,12 @@ If you want to install IPFS in a server environment and offer IPFS as a service,
 
 @[youtube](-SYDlid7Nqs)
 
+# Kubernetes
```

Comment:
Instead of a dedicated section, let's change it to an admonition in the form of a tip as the current instructions aren't based on the user using Kubernetes (which we can modify this in the future).

For "tip" is would be:

```
::: tip
// content
:::